components/site-search/element.js

```diff
@@ -1,20 +1,75 @@
 import { Task } from "@lit/task";
 import { LitElement, html, nothing } from "lit";
 
+import { ifDefined } from "lit/directives/if-defined.js";
+import { unsafeHTML } from "lit/directives/unsafe-html.js";
+
 import { L10nMixin } from "../../l10n/mixin.js";
 
+import "../button/element.js";
+
+// import chevronLeftIcon from "../icon/chevron-left.svg?lit";
+// import chevronRightIcon from "../icon/chevron-right.svg?lit";
+import searchIcon from "../icon/search.svg?lit";
+
 import styles from "./element.css?lit";
+
+/**
+ * @type {{[key: string]: string}}
+ */
+const localeMap = {
+  "en-us": "English (US)",
+  de: "Deutsch",
+  fr: "Français",
+  es: "Español",
+  it: "Italiano",
+  ja: "日本語",
+  ko: "한국어",
+  "pt-br": "Português (BR)",
+  ru: "Русский",
+  "zh-cn": "简体中文",
+  "zh-tw": "繁體中文",
+};
```

Comment:
We should probably centralize this somewhere.

Reply:
Thought the same, but could not think of a proper place. The items in the language switcher are coming out of rari responses afaik, so this is the only place where this data is needed.